src/Browser.ts

```diff
@@ -170,12 +170,11 @@ export class Browser {
 
         const $ = await this.#get$(loginUrl);
 
-        const fkey = $('input[name="fkey"]').val();
+        const fkeyElem = $('input[name="fkey"]');
+        const fkey = fkeyElem.val();
 
         if (typeof fkey === "undefined") {
-            throw new InternalError(
-                "Unable to find fkey element on /users/login"
-            );
+            throw new InternalError(`fkey missing (login): ${fkeyElem.html()}`);
```

Comment:
See my other comment below about dynamic strings as errors.

Reply:
Done! Subclassed `ChatExchangeError` to get a simple implementation of `ScrapingError` with an `html` property and an optional `selector` property instead, dropped the HTML string dumping